scifight/templatetags/scifight_url.py

```diff
@@ -0,0 +1,53 @@
+from django import template
+from scifight import models
+from django.core.urlresolvers import reverse, NoReverseMatch
+
+
+register = template.Library()
+
+_single_arg_views = {
+    models.Participant: ("scifight:participant",   'id'),
+    models.Leader:      ("scifight:leader",        'id'),
+    models.Jury:        ("scifight:jury",          'id'),
+    models.Room:        ("scifight:room",          'id'),
+    models.Fight:       ("scifight:fight",         'id'),
+    models.Problem:     ("scifight:problem",       'problem_num')
+}
+
+
+@register.filter(is_safe=True)
+def scifight_url(model):
+    """
+    This filter simplifies the use of a template.
+    Input parameter is a model object.
+    {{ object | scifight_url }}
+    The filter will return the URL, from the corresponding
+    item in the urls.py. In case of an error NoReverseMatch
+    returns an empty string
+    """
+    def wrapped_reverse(*args, **kwargs):
+        try:
+            return reverse(*args, **kwargs)
+        except NoReverseMatch:
+            return ''
+
+    if isinstance(model, models.Team):
+        if model.slug:
+            kwargs = dict(team_slug=model.slug,
+                          tournament_slug= model.tournament.slug)
```

Comment:
To me, `{ … }` instead of `dict` suits better here. In general, avoid using `dict` unless it gives you strong readability benefits.